deck.js

```diff
@@ -16,20 +16,22 @@ function createDeck() {
     var suits = ['mug','tent','campfire','sleepingbag'];
     for (var i = 0; i < suits.length; i++) {
         var suit = suits[i];
-        for (var j = 1; j < 10; j++) {
-            deck.push(new card(suit, j, j));
+        for (var j = 1; j <= 13; j++) {
+            deck.push(new card(suit, j));
         }
-        deck.push(new card(suit, "J", 10));
-        deck.push(new card(suit, "Q", 10));
-        deck.push(new card(suit, "K", 10));
     }
     return deck;
 }
 
-function card(suit, num, pegValue) {
+function card(suit, rawNum) {
     this.suit = suit;
-    this.num = num;
-    this.pegValue = pegValue;
+    switch (rawNum) {
+        case 11 : this.num = "J"; break;

```

Comment:
Bullshit switch indenting.

Reply:
Ha ha. This looks so nice with inlined cases though.